frontend/src/__tests__/cypress/cypress/pages/loginDialog.ts

```diff
@@ -0,0 +1,20 @@
+import { Modal } from '~/__tests__/cypress/cypress/pages/components/Modal';
+
+class LoginDialog extends Modal {
+  protected testId;
+
+  constructor(title: string, testId: string) {
+    super(title);
+    this.testId = testId;
+  }
+
+  find() {
+    return cy.findByTestId(this.testId);
+  }
+
+  findLoginButton() {
+    return this.findFooter().findByTestId('modal-login-button');
+  }
+}
+
+export const loginDialog = new LoginDialog('Session Expired', 'session-expired-modal');
```

Comment:
No need to parameterize the constructor since you're not going to instantiate different login dialogs:

```suggestion
class LoginDialog extends Modal {

  constructor() {
    super('Session Expired');
  }

  find() {
    return cy.findByTestId('session-expired-modal');
  }

  findLoginButton() {
    return this.findFooter().findByTestId('modal-login-button');
  }
}

export const loginDialog = new LoginDialog();
```